webui/templates/login.html

```diff
@@ -3,21 +3,36 @@
 
 {% block body %}
 
-<div class="container min-vh-100">
+<div class="container pb-5">
+
+  <div class="row">
+  <div class="col-12">
+    <nav aria-label="breadcrumb" id="breadcrumb">
+      <ol class="breadcrumb py-4">
+        <li class="breadcrumb-item pl-3">
+          <a href="{{ url_for('index') }}"><i class="fad fa-home fa-fw mr-2"></i>Home</a>
+        </li>
+        <li class="breadcrumb-item active" aria-current="page">Profile</li>
+      </ol>
+    </nav>
+  </div>
+</div>
 
   <div class="row my-5">
-    <div class="col-12 col-sm-4 offset-sm-4">
+    <div class="col-12 col-md-6 offset-md-3">
       <div class="card">
         <img src="{{ url_for('static', filename='orcid.png') }}" class="card-img-top" alt="ORCID Logo">
-        <div class="card-body">
-          <h2 class="text-center">Sign in with ORCID<h2>
-              <p class="lead text-center my-4">To create and save carrels, you must authenticate with your ORCID ID.</p>
-              <p class="text-center lead"> If you don't have an ORCID account, you may <a href="https://orcid.org/" target="_blank">register for free</a> at the ORCID website.</p>
-              <form method="post" class="my-4">
-                <div class="form-group  text-center">
-                  <input class="btn btn-primary text-light" type="submit" value="Sign In">
-                </div>
-              </form>
+        <div class="card-body px-4">
+
+          <form method="post" class="my-4">
+            <div class="form-group  text-center">
+              <input class="btn btn-primary btn-lg text-light" type="submit" value="Sign In">
+            </div>
+          </form>
+
+          <p class="lead mt-5 mb-3">To create and save carrels, you must sign in. We encourage users to login with an ORCID ID. If you don't have an ORCID account, you may <a href="https://orcid.org/" target="_blank">register for free</a> at the ORCID website.</p>
```

Comment:
I find this text confusing. From the distant reader's perspective everyone _must_ have an ORCID. it is only when authenticating with orcid that people could use a linked google account, or whatnot.